src/components/dashboard/DashboardPage.jsx

```diff
@@ -1,34 +1,28 @@
 import React, { useContext } from 'react';
 import { Helmet } from 'react-helmet';
-import { useQueryClient } from '@tanstack/react-query';
-import { Alert, Container, Tabs } from '@openedx/paragon';
+import {
+  Alert, Container, Tabs, useToggle,
+} from '@openedx/paragon';
 import { AppContext } from '@edx/frontend-platform/react';
 import { FormattedMessage, useIntl } from '@edx/frontend-platform/i18n';
-
 import { IntegrationWarningModal } from '../integration-warning-modal';
 import SubscriptionExpirationModal from './SubscriptionExpirationModal';
 import { useDashboardTabs } from './data';
-import { querySubscriptions, useEnterpriseCustomer, useSubscriptions } from '../app/data';
+import { useEnterpriseCustomer, useSubscriptions } from '../app/data';
 import BudgetExpiryNotification from '../budget-expiry-notification';
 import ExpiredSubscriptionModal from '../expired-subscription-modal';
 
 const DashboardPage = () => {
   const intl = useIntl();
-  const queryClient = useQueryClient();
   const { authenticatedUser } = useContext(AppContext);
   const userFirstName = authenticatedUser?.name?.split(' ').shift();
-
+  const [shouldShowLicenseActivationSuccessMessage, , close] = useToggle(!!sessionStorage.getItem('shouldShowActivationSuccessMessage'));
```

Comment:
Name mismatch in the session storage key, `shouldShowLicenseActivationSuccessMessage` is used now. Optionally consider creating a constant to re-use if it makes it less prone to mismatches?